### DISCUSSION TITLE
Custom REPL demo is missing utils.py

### DISCUSSION THREAD

**Discussion by pedrokarneiro** [pyscript/pyscript#1482] [General]
### Custom REPL demo is missing utils.py in the 'View code' tab.

Problem:

If you try to copy the solution, a 403 error will come up due to this absence.

Solution:

Add a **utils.py** to your solution with the following code:

`
from datetime import datetime as dt

def format_date(dt_, fmt="%m/%d/%Y, %H:%M:%S"):
    return f"{dt_:{fmt}}"

def now(fmt="%m/%d/%Y, %H:%M:%S"):
    return format_date(dt.now(), fmt)

def remove_class(element, class_name):
    element.element.classList.remove(class_name)

def add_class(element, class_name):
    element.element.classList.add(class_name)
`

**Comment by WebReflection** [comment]
Thanks, this is being tackled in here: https://github.com/pyscript/pyscript/pull/1485
